src/rvi.c

```diff
@@ -1050,6 +1050,38 @@ TRviHandle rviInit ( char *configFilename )
     return NULL;
 }
 
+
```

Comment:
I did it because it requiters less changes and no need to write additional internal function. But maybe You are right, I will rewrite it without custom file reading.